### DISCUSSION TITLE
Blog comment on From Sunday night blues to Monday morning cheers - The magic of company culture

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#2103] [Docs/blog comments]
# Blog comment on From Sunday night blues to Monday morning cheers - The magic of company culture

How do you explain culture, how do you put a feeling into words...

https://weaviate.io/company/playbook/from-sunday-nights-blues-to-monday-morning-cheers-the%20magic-of-company-culture

**Comment by GirijaDogra** [comment]
This is an amazing read, Thanks Jessie !
